apps/docs/content/components/Grid/index.mdx

```diff
@@ -0,0 +1,286 @@
+---
+title: Grid
+description: Use the grid component to create flexible and responsive grid-based layouts.
+---
+
+import {Box, Text} from '@primer/react'
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![The grid component applied to a section with different elements that spans in columns](https://github.com/primer/brand/assets/912236/6cd2805e-0148-4a1d-b373-07acd1aa7f51)
+
+## Usage
+
+Use the grid component to define a layout that maintains consistent element positioning and sizing across pages. The grid is based on a 12-column layout and uses vertical and horizontal gaps to separate content units consistently[^1].
+
+Use the grid for elements with related content or similar purposes. E.g. A list of [cards](/components/Card) or a "contact us" form.
+
+In most cases, we recommend using the same column span for each element to create a consistent layout.
+
+[^1]: The grid component is implemented using the [CSS Grid Layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) specification.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/912236/b4870b2e-5cf0-440c-9f51-0b1f81258b9b"
+      alt=""
+    />
+    <Caption>
+      Use the same column span for each content unit to create a consistent
+      layout and spacing.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/912236/0703ba43-7f47-4ad1-bc1d-020bc095e362"
+      alt=""
+    />
+    <Caption>
+      Don't use different column spans when grouping the same element type.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+In some cases, you may need specific amount of columns or rows instead of an even distribution. Use different column spans when creating a visual hierarchy between items or when defining a page or section layout structures. E.g. A section with an [image](/components/Image) and [text](/components/text), a form with inputs and text areas, or a layout with main content and a sidebar.
+
+![](https://github.com/primer/brand/assets/912236/bc442c41-3546-4341-bab3-1c649ccfe7d2)
```

Comment:
Could we add some alt text in here?

Alternatively, put this as another `do` guidelines, as it currently looks a little unclear when scanning the page whether this is good or bad, as it's adjacent to a do/don't block but unlabelled.

![Screenshot 2023-07-12 at 11 40 37](https://github.com/primer/brand/assets/13340707/b398c86b-be5c-42c2-ad6d-e3dc5e845808)

Reply:
Added a do-don't example with it